RELEASE.md

```diff
@@ -8,87 +8,60 @@ Versioning
 
 Follow [semantic versioning 2.0.0][semver] to choose the new version number.
 
-Prepare change log entries
---------------------------
+The format of release notes
```

Comment:
Release notes will be generated by release API automatically and the format of it doesn't look guaranteed. We only need to confirm that the output is correct regardless of the format.

Reply:
This remark was wrong. Helm releases are edited manually.

Reply:
What I tried to say here is that we need to categorize PRs into categories (Added, Changed, Fixed, ...) maually.
So, I showed how the final result should look like.

Reply:
I fixed it.